package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Brave Core is a set of changes, APIs and scripts used for customizing Chromium to make Brave.",
   "main": "index.js",
   "scripts": {
+    "help": "node ./build/commands/scripts/commands.js --help",
```

Comment:
This should also go in a separate PR as it is not directly related. Having a lot of unrelated changes makes the PR harder to understand and also harder to look back and see why things changed and reverts also become more complicated

Reply:
moved to separate PR: https://github.com/brave/brave-core/pull/30363